unity-renderer/Assets/DCLPlugins/ECS7/Systems/Helpers/ECSSystemsReferencesContainer.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using DCL.Controllers;
+using DCL.ECSRuntime;
+
+namespace ECSSystems.Helpers
+{
+    public static class ECSSystemsReferencesContainer
+    {
+        public static IReadOnlyList<IParcelScene> loadedScenes;
+        public static IECSComponentWriter componentsWriter;
```

Comment:
This should be in the DataStore

Reply:
That class is just for containing references that are needed for a lot of other systems.
Main intention is to avoid accessing DataStore directly, so, for example `loadedScenes` there contain a reference to a DataStore here: https://github.com/decentraland/unity-renderer/blob/badd9b2ce38a9255208771c78bfa56c69f593f4f/unity-renderer/Assets/DCLPlugins/ECS7/ECS7Plugin.cs#L37
because, when we start to add up systems, it start to become noticeable the performance loss on accessing the DataStore directly on every system, on every update loop

Reply:
The performance loss (I would love to see numbers about the actual loss) comes from the redirection between the instance, the sub data store and then the data.

You can circumvent the issue easily with dependency injection. Your system that would do `ReferencesContainer` now simply accepts on its constructor an instance to whatever it needs and altough that stuff lives in the DataStore is never accessed through it. 

You avoid the performance loss, keep the design coherent with the rest of the project and also avoid the pitfalls associated to static classes with state.

Reply:
Yes, DataStore do a lookup at a dictionary and then a cast, which is not convenient to do multiple times in every frame.
In any case, I've just removed the static class that worked as reference container here: https://github.com/decentraland/unity-renderer/pull/2663/commits/3017565965a96310df50cf55792670881fb18a68